src/components/Dialog/Dialog.css

```diff
@@ -190,15 +188,3 @@
 [data-test-id='upload-source-select-dropdown'] {
   margin-right: 3px;
 }
-
-@media only screen and (min-width: 481px) {
-  .ix-container {
-    padding: 40px;
-  }
-}
-
-@media only screen and (min-width: 1025px) {
-  .ix-container {
-    padding: 40px 64px;
-  }
-}
```

Comment:
This will need to be reintroduced. Removing it, for now, to make the padding consistent across all the elements.

I will circle back and add break-points-specific padding in a day or two. It will not be merged before it's addressed.

Reply:
Just want to make sure this gets addressed before we merge.

Reply:
Discussed offline. Will fix in future PR.